src/vs/base/browser/ui/scrollbar/scrollableElement.ts

```diff
@@ -625,5 +625,15 @@ function resolveOptions(opts: ScrollableElementCreationOptions): ScrollableEleme
 		result.className += ' mac';
 	}
 
+	if (typeof process === 'object' && process.env.VSCODE_ALWAYS_SHOW_SCROLLBARS === '1') {
```

Comment:
We do not use environmental variables for configuration. IMHO this should be a setting. There are already "hidden" editor specific options, `editor.scrollbar.horizontal` and `editor.scrollbar.vertical`.

But I think we need a new global setting like `scrollbar.visible: "auto" | "never" | "always"` and then this setting needs to be adopted in all users of the scrollableElement widget.